PowerAuth2.podspec

```diff
@@ -1,17 +1,18 @@
 Pod::Spec.new do |s|
 	# General information
+    s.cocoapods_version = '>= 1.9'
```

Comment:
Tabs vs. spaces inconsistency...

Reply:
I converted all podspecs to spaces (ruby)